### PR TITLE
Docs: Add cypress extension

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

Doc for the new Cypress extension feature. Reference:  #1921 

Preview: https://faststore-site-git-docs-cypress-extension-faststore.vercel.app/docs/faststore-config-options/custom-cypress-test

### CODE REVIEW

File: apps/site/pages/docs/fastore-config-options/custom-cypress-test.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+Cypress extensions enhance the testing capabilities of your Faststore project by enabling the incorporation of custom Cypress tests alongside the default tests provided by Faststore.
+
+By default, Faststore includes standard integration tests for your store and with `enableCypressExtension`, you can integrate custom integration tests into a dedicated Cypress root folder within your store's codebase. 
+This integration ensures that both the conventional integration tests and your personalized tests are executed seamlessly as part of the continuous integration (CI) pipeline.
+
+
+## Step 1: Enabling `enableCypressExtension` 
+
+1. Open the `faststore.config.js` file.
+2. Find the `enableCypressExtension` paramether and set it to `true`.
```
Comment (Gmantiqueira) [inline apps/site/pages/docs/fastore-config-options/custom-cypress-test.mdx:29]: We'll not let the `experimental` object there as default, it's like a "secret feature". Can we change that, please? 😁 They need to put the object `experimental` and then `enableCypressExtension` to true.

File: apps/site/pages/docs/fastore-config-options/custom-cypress-test.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (Gmantiqueira) [inline apps/site/pages/docs/fastore-config-options/custom-cypress-test.mdx:1]: Folder typo, `fastore-...` to `faststore`.

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+Cypress extensions enhance the testing capabilities of your Faststore project by enabling the incorporation of custom Cypress tests alongside the default tests provided by Faststore.
+
+By default, Faststore includes standard integration tests for your store and with `enableCypressExtension`, you can integrate custom integration tests into a dedicated Cypress root folder within your store's codebase. 
+This integration ensures that both the conventional integration tests and your personalized tests are executed seamlessly as part of the continuous integration (CI) pipeline.
+
+
+## Step 1: Enabling `enableCypressExtension` 
+
+1. Open the `faststore.config.js` file.
+2. Add the `experimental` object with the `enableCypressExtension` as it parameter.
```
Comment (Gmantiqueira) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:29]:
```suggestion
2. Add the `experimental` object with the `enableCypressExtension` as it property.
```

**Review by Gmantiqueira — APPROVED** [review]

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+Cypress extensions enhance the testing capabilities of your Faststore project by enabling the incorporation of custom Cypress tests alongside the default tests provided by Faststore.
+
+By default, Faststore includes standard integration tests for your store and with `enableCypressExtension`, you can integrate custom integration tests into a dedicated Cypress root folder within your store's codebase. 
+This integration ensures that both the conventional integration tests and your personalized tests are executed seamlessly as part of the continuous integration (CI) pipeline.
+Learn how to add custom integrations in the following steps.
+
+## Before you start
+
+## Before You Begin
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:28]:
```suggestion
```

File: apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+The `enableCypressExtension` experimental feature flag allows you to utilize Cypress extensions within your FastStore project for running customized Cypress tests seamlessly.
+
+Cypress extensions enhance the testing capabilities of your Faststore project by enabling the incorporation of custom Cypress tests alongside the default tests provided by Faststore.
+
+By default, Faststore includes standard integration tests for your store and with `enableCypressExtension`, you can integrate custom integration tests into a dedicated Cypress root folder within your store's codebase. 
+This integration ensures that both the conventional integration tests and your personalized tests are executed seamlessly as part of the continuous integration (CI) pipeline.
+Learn how to add custom integrations in the following steps.
+
+## Before you start
+
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx:27]:
```suggestion
```

**Pull request merged**